adapters/mobfoxpb/mobfoxpb.go

```diff
@@ -0,0 +1,137 @@
+package mobfoxpb
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/buger/jsonparser"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type MobfoxpbAdapter struct {
+	URI string
+}
+
+// NewMobfoxpbBidder Initializes the Bidder
+func NewMobfoxpbBidder(endpoint string) *MobfoxpbAdapter {
+	return &MobfoxpbAdapter{
```

Comment:
Please consider adding validation of the endpoint URL here before returning the adapter instance. Something like:
```
endpointURL, err := url.ParseRequestURI(endpoint)
if err != nil {
	glog.Fatalf("invalid endpoint provided for Mobfox: %s, error: %v", endpoint, err)
	return nil
}
return &MobfoxpbAdapter{
	URI: endpointURL.String(),
}
```

Reply:
Not sure how to handle this case properly, CI, when tests prebid-server it also tests adapters with empty config, so if i'm returning nil from adapter end printing error message - testing stops with failed status

Reply:
@mobfxoHB Can I ask which CI are you referring to here?

Reply:
Github circle ci, after failed commit - i've reverted changes

Reply:
@mobfxoHB That looks like something specific to your CI configuration. Perhaps you can change that on your end? Because in case of invalid endpoints, it is better to validate and fail early than run into issues while processing requests.

Reply:
We have no CI on our side - test are beeing done only by this one:
Validate / Go (1.13.x, ubuntu-18.04) (pull_request)
Validate / Go (1.14.x, ubuntu-18.04) (pull_request)
Validate / Go (1.15.x, ubuntu-18.04) (pull_request)
And there , i think can not be invalid URL - we use only default one